pagopa/io-app-design-system
# [IOAPPX-403] Action in banner is not announced by screen reader on Android

## Short description
This PR resolves an accessibility issue where the `action` in the `Banner` component was not properly announced by Screen Readers on Android devices. The changes introduced propose a solution while ensuring that existing functionality remains unaffected

## List of changes proposed in this pull request
- Introduced `fallbackAccessibilityLabel`, a dynamically generated label combining the `title`, `content`, and `action` if they are present.
-  If `accessibilityLabel` is not provided, `fallbackAccessibilityLabel` is used as a default

## How to test
To test the changes, import the updated library containing the modifications into the IO app and ensure that the Screen Reader announces the title, content, and action accurately, even when one of these elements is missing. 


**Warning: This solution has been tested on an Android device and on an iOS simulator. Remains to be tested on iOS device**
